<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(217, 217, 217);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>426</y>
      <width>280</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(155, 155, 155);
	border: none;
	padding-top: 5px;
	border-radius: 40px;
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TheGodGame">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>50</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;The God game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Leave">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>160</y>
      <width>170</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="UI/resources/Images.qrc">:/Images/Images/leave.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Fon">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>911</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);</string>
    </property>
    <property name="text">
     <string>`</string>
    </property>
   </widget>
   <zorder>Fon</zorder>
   <zorder>PlayButton</zorder>
   <zorder>TheGodGame</zorder>
   <zorder>Leave</zorder>
  </widget>
 </widget>
 <resources>
  <include location="UI/resources/Images.qrc"/>
 </resources>
 <connections/>
</ui>
